test/test_helper.exs

```diff
@@ -16,6 +16,12 @@ defmodule TestPool do
   def pool_type, do: DBConnection.ConnectionPool
 end
 
+defmodule TestDatabaseObserver do
+  def db_connection_reset_callback(_state) do
+    IO.puts("Callback: track connection reset on APM")
```

Comment:
Will this cause an error in credo checking?  Maybe we can either make this a Logger or find another way that does not use IO.puts?

Reply:
Credo is actually ok with it:

```elixir
$ mix credo test/test_helper.exs
Checking 1 source file ...

Please report incorrect results: https://github.com/rrrene/credo/issues

Analysis took 0.3 seconds (0.1s to load, 0.1s running 52 checks on 1 file)
5 mods/funs, found no issues.
```

Anyway, let me replace it by a simple `:ok` atom.